content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/zebra.rst

```diff
@@ -0,0 +1,235 @@
+
+Next, click the :icon:`fa-code` :guilabel:`Qweb Views` smart button, and choose the desired label
+:doc:`view <../../../../../developer/reference/user_interface/view_records>`.
+
+.. figure:: zebra/qweb-views.png
+   :align: center
+   :alt: Show Qweb smart button on the Lot and Serial Number (ZPL) report.
+
+   **Lot and Serial Number (ZPL)** report, highlighting the Qweb smart button.
+
+On the resulting view form, go to the :guilabel:`Architecture` tab to view the |ZPL| code.
+
+.. important::
+
+   To ensure the customization is **not** overwritten during an update, click the :icon:`fa-bug`
+   :guilabel:`(bug)` icon on the view page. Then, select the :guilabel:`View Metadata` option from
+   the resulting drop-down menu, in order to open the :guilabel:`View Metadata` pop-up window. Then,
+   ensure the :guilabel:`No Update` field is set to :guilabel:`true (change)`. Click :guilabel:`Ok`
+   to exit the :guilabel:`View Metadata` pop-up window.
+
+.. image:: zebra/architecture.png
+   :align: center
+   :alt: Architecture tab in the view.
+
+.. _inventory/shipping_receiving/margin:
+
+Adjust margin
+=============
+
+Text gets cut off from standard |ZPL| labels printed in Odoo when the line exceeds fifty-five
+characters. To fit long product names, or lot numbers, on a single line, adjust the margin.
+
+To begin, navigate to the :ref:`ZPL code of the label <inventory/shipping_receiving/zpl-view>` in
+the :guilabel:`Architecture` tab. In the |ZPL| code for product labels, look for the `^FT` command,
+which specifies where to start placing the text, or graphic element, on the label. The two numbers
+immediately following `^FT` define the x-coordinate and y-coordinate in dots (:dfn:`similar to
+pixels for printers`) from the left and top margins.
+
+.. important::
+   When customizing lot/serial number labels, look for the `^FO` command, instead of `^FT`.
+
+.. example::
+
+    The following is an example where the product's name gets cut off with Odoo's default |ZPL|
+    formatting. In the **Fixed** tab, the x-coordinate of the starting position of the label is
+    changed from `^FT100,80` to `^FT0,80`, to fit the entire name.
+
+.. tabs::
+
+    .. tab:: Default
+
+        .. image:: zebra/default-margin.png
+           :align: center
+           :alt: Example barcode label with the product name cut off.
+
+        **Code**:
+
+        .. code-block:: xml
+
+            ^XA^CI28
+            ^FT100,80^A0N,40,30^FD[E-COM11] Cabinet with Doors (wood: Cherry, handles: brass)^FS
+            ...
+            ^XZ
+
+    .. tab:: Fixed
+
+        .. image:: zebra/fixed-margin.png
+          :align: center
+          :alt: Example barcode label with the product name margin adjusted to the left.
+
+        **Code**:
+
+        .. code-block:: xml
+
+            ^XA^CI28
+            ^FT0,80^A0N,40,30^FD[E-COM11] Cabinet with Doors (wood: Cherry, handles: brass)^FS
+            ...
+            ^XZ
```

Comment:
Alignment and suggestion to use the term "Modified" instead of "Fixed" since [tab headings are not translated](https://www.odoo.com/documentation/17.0/contributing/documentation/rst_cheat_sheet.html#content-tabs)
```suggestion
.. tabs::

   .. tab:: Default

      .. image:: zebra/default-margin.png
         :align: center
         :alt: Example barcode label with the product name cut off.

      **Code**:

      .. code-block:: xml

         ^XA^CI28
         ^FT100,80^A0N,40,30^FD[E-COM11] Cabinet with Doors (wood: Cherry, handles: brass)^FS
         ...
         ^XZ

   .. tab:: Modified

      .. image:: zebra/fixed-margin.png
         :align: center
         :alt: Example barcode label with the product name margin adjusted to the left.

      **Code**:

      .. code-block:: xml

         ^XA^CI28
         ^FT0,80^A0N,40,30^FD[E-COM11] Cabinet with Doors (wood: Cherry, handles: brass)^FS
         ...
         ^XZ
```